dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1777,7 +1790,7 @@ def _get_num_zeros(
         """
         num_zeros_value = (df_series == 0).sum()
         subset_properties["num_zeros"] = num_zeros_value
-        self.num_zeros = self.num_zeros + num_zeros_value
+        self.num_zeros = int(self.num_zeros + num_zeros_value)
```

Comment:
Cast added to adhere to type specification in numeric stats mixin attribute initialization (avoids setting to np.int64)